src/components/flyout/__tests__/OverlayHeader.test.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import sinon from 'sinon';
+
+import FlyoutContext from '../FlyoutContext';
+import OverlayHeader from '../OverlayHeader';
+
+const sandbox = sinon.sandbox.create();
```

Comment:
We've been moving away from sinon. Can we use Jest for any mocks or spies we need?